frontend/cypress/integration/account-details-navigation.spec.ts

```diff
@@ -31,7 +31,7 @@ context("Account Details navigation", () => {
               if (mixedcaseAccountId === mixedcaseAccountId.toLowerCase()) {
                 expect(resp.status).to.eq(200);
               } else {
-                expect(resp.status).to.eq(301);
+                expect(resp.status).to.eq(308);
```

Comment:
`Next.js` seems to be using 308 instead of 301, and the only difference is that 301 allows method to be changed to `GET` and 308 does not.
Doesn't seems relatable to this case.